src/ReTestItems.jl

```diff
@@ -693,7 +709,7 @@ function include_testfiles!(project_name, projectfile, paths, ti_filter::TestIte
             if !(is_testsetup_file(filepath) || (is_test_file(filepath) && is_requested(filepath, paths)))
                 continue
             end
-            fpath = relpath(filepath, project_root)
+            fpath = nestedrelpath(filepath, project_root)
             file_node = FileNode(fpath, ti_filter; report, verbose=verbose_results)
```

Comment:
i think i didn't do this because i thought some package was using `FileNode` directly (despite it being internal)

but now i can't find such package so maybe would be okay to change